icon_validator/rules/workflow_validators/workflow_use_case_validator.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from icon_plugin_spec.plugin_spec import KomandPluginSpec
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class WorkflowUseCaseValidator(KomandPluginValidator):
+    @staticmethod
+    def get_approved_usecase_tags(response_json: dict) -> [str]:
+        approved_usecases = []
+        for result in response_json["results"]:
+            if result["type"] == "usecase":
+                approved_usecases.append(result["name"])
+
+        return approved_usecases
+
+    @staticmethod
+    def get_approved_usecase_tags_with_paging() -> [str]:
+        approved_usecases = []
+        query = "types=usecase"
+        for i in range(0, 9999):
+            response = requests.get(url=f"https://extensions-api.rapid7.com/v2/public/tags?first=1000&{query}")
+            response_json = response.json()
+            approved_usecases.extend(WorkflowUseCaseValidator.get_approved_usecase_tags(response_json))
+            if not response_json["pageInfo"]["hasNextPage"]:
+                break
+
+            query = f"after={response_json['pageInfo']['endCursor']}"
```

Comment:
```suggestion
        params = {
            "first": 1000
            "types": "usecase"
        }
        
        has_next = True
        while has_next:
            response = requests.get(url=f"https://extensions-api.rapid7.com/v2/public/tags", params=params)
            response_json = response.json()
            approved_usecases.extend(WorkflowUseCaseValidator.get_approved_usecase_tags(response_json))
            if not response_json["pageInfo"]["hasNextPage"]:
                has_next = False

            params["after"] = response_json['pageInfo']['endCursor']
```
Cleaned up the request a bit - you can pass dictionaries into the `params` arg in requests.get() to use query parameters in a cleaner way. Also switched the for loop to a while. I highly doubt we'd ever paginate 10000 times, but you never know 🤷🏻 . Also fixed the query update on line 29 - you were clearing out the existing params which was yielding an HTTP 400 when I tried it in the browser.

Give this code a shot